rapids-cmake/cpm/nvtx3.cmake

```diff
@@ -114,6 +114,13 @@ function(rapids_cpm_nvtx3)
     endif()
   endif()
 
+  if(NOT TARGET nvtx::nvtx3-c AND TARGET nvtx3-c)
```

Comment:
The docs are using nvtx3:: but the logic says nvtx::. Which do we want?

Reply:
The tests are also using `nvtx3::`.

Reply:
yeah we should use `nvtx3::`